portfolio/src/main/java/com/google/sps/servlets/ListComment.java

```diff
@@ -69,4 +71,16 @@ private int getCommentCount(HttpServletRequest request){
         }
         return numComments;
     }
+
+    private String getUserNickname(String id) {
```

Comment:
A thing to consider for a future PR: this method is being repeated among two classes. It would be optimal if we could extract it to a common class and share it among the callers.